studio/localStores/sqlEditor/SqlEditorStore.js

```diff
@@ -19,6 +18,10 @@ export function useSqlEditorStore(ref, meta) {
   return store
 }
 
+export function createSqlEditorStore(ref, meta) {
+  return new SqlEditorStore(ref, meta)
```

Comment:
any reason why we need this? seems like its repeating `useSqlEditorStore`

Reply:
Yep I can just use `useSqlEditorStore`, I created another one because I felt dirty using a 'hook' in a non-hook way lol

Reply:
curious to know what you mean by using a hook in a non-hook way

i think i might have an idea of what you mean haha, but i guess might just be a matter of naming then (e.g the largely used `useStore`) we typically just use this as the entry point to accessing the mobx store

Reply:
Like when I see a function that starts with `use`, I assume that I have to call it in the context of reacts rendering (so it can access context etc), but I was calling it inside a `useEffect`. I've reverted back to the actual hook now though :)

Reply:
ahh okay gotcha - yeah think it'll be great (if when we have the time and headspace) to sit down discuss and lay down some guidelines on how to do things around in studio. i do admit that its still quite messy atm

e.g I tried to work up something with a couple of READMEs (interfaces folder, components folder, etc)